test/format/ops.c

```diff
@@ -230,12 +230,15 @@ tinfo_teardown(void)
 static void
 rollback_to_stable(WT_SESSION *session)
 {
+    char cfg[512];
+
     /* Rollback-to-stable only makes sense for timestamps. */
     if (!g.transaction_timestamps_config)
         return;
 
-    /* Rollback the system. */
-    testutil_check(g.wts_conn->rollback_to_stable(g.wts_conn, NULL));
+    /* Rollback the system using up to 10 threads. */
+    testutil_snprintf(cfg, sizeof(cfg), "treads=%" PRIu64, WT_MIN(10, GV(RUNS_THREADS)));
+    testutil_check(g.wts_conn->rollback_to_stable(g.wts_conn, cfg));
```

Comment:
My only worry here:

- We are no longer testing a NULL config
- We are not testing a config with 0 threads

Should I add some randomness to cover the scenarios above?

Reply:
Something more simple would be to do:
```
mmrand(&g.extra_rnd, 1, 10)
```
And just assign the value to the `threads` but we are still not covering the NULL config.

Reply:
The NULL config also previously used 4 worker threads by default.

Reply:
Added the NULL case in ddc3607b600a4080ce4dde2ac747fda52af1c9e0. Let me know what you think @kommiharibabu